PURL-TYPES.rst

```diff
@@ -59,6 +59,26 @@ apk
       pkg:apk/alpine/curl@7.83.0-r0?arch=x86
       pkg:apk/alpine/apk@2.12.9-r3?arch=x86
 
+bazel
+-----
+``bazel`` for Bazel modules as specified at ``https://bazel.build/external/module``:
+
+- The default repository is the Bazel Central Registry (BCR) ``https://bcr.bazel.build``.
+- The ``name`` is the module name.
+- The ``version`` is the module version.
+- The optional ``repository_url`` can be used to specify the URL of an
+  alternative registry.
+- The optional ``subpath`` can refer to a particular Bazel package in the
```

Comment:
Can you clarify for me what a package means here? From https://bazel.build/concepts/build-ref#packages I get that this is the path to a directory relative to the root of a module, where the directory must contain a BUILD.bazel file and the root of the module a MODULE.bazel file. Is this correct? Is there something we should add to the definition?

Reply:
That's pretty much it (`BUILD` files are also allowed). It's one of the top two core concepts of Bazel but is also indistinguishable from a "path from the root", so I'm not sure whether additional details should be added to the spec. What would you be looking for?

Reply:
I needed to understand. I want to make sure that `can refer to a particular Bazel package in the module` is not ambiguous.

Reply:
I can clarify this, but my understanding is that subpath should be a valid file path for all purls and a label with prefix `//` is not. What do you think?

Reply:
`//foo/bar` is not a valid subpath. PURL implementations are required to read `//foo/bar` as `foo/bar`.

Reply:
Meh...

Should we at least add a link to https://bazel.build/concepts/labels#package-names then? Even if we don't enforce the restrictions

Reply:
Sure, added the link